README.md

```diff
@@ -5,79 +5,86 @@ OpenShift Application Platform
+  * Roll out modifications to software stacks to your entire organization in a controlled fashion
+  * Integration with your existing authentication mechanisms, including LDAP, Active Directory, and public OAuth providers such as GitHub
+* Multi-tenancy support, including team and user isolation of containers, builds and network communication
   * Allow developers to run containers securely with fine-grained controls in production
   * Limit, track, and manage the developers and teams on the platform
 * Integrated Docker registry, automatic edge load balancing, cluster logging, and integrated metrics
 
 **Learn More:**
 
 * **[Public Documentation](http://docs.openshift.org/latest/welcome/index.html)**
-* The **[Trello Roadmap](https://ci.openshift.redhat.com/roadmap_overview.html)** covers the epics and stories being worked on (click through to individual items)
+  * **[API Documentation](http://docs.openshift.org/latest/rest_api/openshift_v1.html)**
 * **[Technical Architecture Presentation](https://docs.google.com/presentation/d/1Isp5UeQZTo3gh6e59FMYmMs_V9QIQeBelmbyHIJ1H_g/pub?start=false&loop=false&delayms=3000)**
 * **[System Architecture](https://github.com/openshift/openshift-pep/blob/master/openshift-pep-013-openshift-3.md)** design document
-* **[API Documentation](http://docs.openshift.org/latest/rest_api/openshift_v1.html)**
+* The **[Trello Roadmap](https://ci.openshift.redhat.com/roadmap_overview.html)** shows work (epics and user stories) that are planned or currently underway (click through to individual items)
 
 For questions or feedback, reach us on [IRC on #openshift-dev](https://botbot.me/freenode/openshift-dev/) on Freenode or post to our [mailing list](https://lists.openshift.redhat.com/openshiftmm/listinfo/dev).
 
-
 Getting Started
 ---------------
 
 ### Installation
 
-If you have downloaded the client tools, place the included binaries in your PATH.
-
-* For a quick install of Origin, see the [Getting Started Install guide](https://docs.openshift.org/latest/getting_started/administrators.html).
-* For an advanced installation using [Ansible](https://github.com/openshift/openshift-ansible), follow the [Advanced Installation guide](https://docs.openshift.org/latest/install_config/install/advanced_install.html)
+* If you intend to develop applications to run on an existing installation of the OpenShift platform, you can download the client tools and place the included binaries in your `PATH`
+* For local development/test or product evaluation purposes, we recommend using the quick install as described in the [Getting Started Install guide](https://docs.openshift.org/latest/getting_started/administrators.html).
+* For production environments, we recommend using [Ansible](https://github.com/openshift/openshift-ansible) as described in the [Advanced Installation guide](https://docs.openshift.org/latest/install_config/install/advanced_install.html)
```

Comment:
We should be consistent in use of the fullstop at the end of each bulletpoint item.